internal/databases/greenplum/query_runner.go

```diff
@@ -168,6 +168,21 @@ SELECT pg_is_in_backup(), -1;
 `
 }
 
+// TryGetLock tries to take advisory lock
+func (queryRunner *GpQueryRunner) TryGetLock() (err error) {
+	conn := queryRunner.Connection
+	var lockFree bool
+	err = conn.QueryRow("SELECT pg_try_advisory_lock('2131121')").Scan(&lockFree)
```

Comment:
Maybe something more semantically loaded?

Reply:
changed it to hashtext('gp_backup')